### DISCUSSION TITLE
Can we play pcm data in this ?

### DISCUSSION THREAD

**Discussion by arunk98475** [schreibfaul1/ESP32-audioI2S#534] [Q&A]
I have 8Khz * 8bit PCM data stream,  which is coming from PC as small data chunks through serial. Is there is any way to input this buffer as audio buffer?

**Comment by kaloprojects** [comment]
Hi, just a comment on '8bit' i assume 8bit waves (from PC) can't be played in moment with @schreibfaul1 audio.h library (but not sure on this, at least i was not succesfull when i real 8bit wav files from PC). But @schreibfaul1 might confirm maybe). I added some more details in my issue post: https://github.com/schreibfaul1/ESP32-audioI2S/issues/786

addon: 8Khz * **16b**it PCM works well (i tested with 'audio.connecttoFS()', also the 8Khz worked well)
